charts/nocodb/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: 0.100.2
```

Comment:
can this value be dynamic or point to latest version?

Reply:
what exactly do you mean by "dynamic"? It has to have some value of course, you cannot let helm dynamically fetch this. You could do "latest", but then if something changes in the application it may break older chart versions.
You can of course use github actions to replace this value before publishing the chart somewhere.

Reply:
I've very basic idea about helm. Possibly wing's question was to send the version from command line or could it be wired to be default set image.tag ? like `helm install noco-helm path/to/chart --set image.tag=0.xyz` ?